### PR TITLE
Removed opener from static pages & modal windows

### PR DESCRIPTION
**Pull request by bennyn**

### CODE REVIEW

File: app/page/template/modals.htm [hunk 1]
```diff
@@ -264,7 +264,7 @@
       </div>
       <div class="modal-text">
         <span data-bind="l10n_text: z.string.modal_session_reset_message_1"></span>
-        <a data-bind="l10n_href: z.string.url_support_contact_bug, l10n_text: z.string.modal_session_reset_message_link" target="_blank"></a>
+        <a rel="nofollow noopener noreferrer" data-bind="l10n_href: z.string.url_support_contact_bug, l10n_text: z.string.modal_session_reset_message_link" target="_blank"></a>
```
Comment (lipis) [inline app/page/template/modals.htm]: as it's less important and closely tied with the `target` maybe putting it just before `target`?

Comment (bennycode) [reply app/page/template/modals.htm]: If you ask me I would just go and sort those properties.

Comment (lipis) [reply app/page/template/modals.htm]: So in that particular case we are both correct :) and as long as the `class` is first I'm fine with it.. Also that: http://codeguide.co/#html-attribute-order (by importance)

Comment (bennycode) [reply app/page/template/modals.htm]: Ok, changed!

File: app/page/template/partials/template-message.htm [hunk 2]
```diff
@@ -39,7 +39,7 @@
   <div class="message-body message-location-body">
     <div class="message-location-icon icon-location"></div>
     <div class="message-location-label" data-bind="text: name"></div>
-    <a target="_blank" class="label-xs text-theme" data-bind="attr: {href: link_src}, l10n_text: z.string.conversation_location_link"></a>
+    <a rel="nofollow noopener noreferrer" target="_blank" class="label-xs text-theme" data-bind="attr: {href: link_src}, l10n_text: z.string.conversation_location_link"></a>
```
Comment (lipis) [inline app/page/template/partials/template-message.htm]:
``` html
<a class="label-xs text-theme" data-bind="attr: {href: link_src}, l10n_text: z.string.conversation_location_link" rel="nofollow noopener noreferrer" target="_blank"></a>
```

File: app/page/template/warning.htm [hunk 3]
```diff
@@ -13,7 +13,7 @@
         <div class="warning-bar warning-bar-feature bg-theme">
           <div class="warning-bar-message">
             <span data-bind="l10n_text: z.string.warning_permission_denied_camera"></span>&nbsp;
-            <a class="warning-bar-link" target="_blank" data-bind="l10n_href: z.string.url_support_camera_access_denied, l10n_text: z.string.warning_learn_more"></a>
+            <a class="warning-bar-link" rel="nofollow noopener noreferrer" target="_blank" data-bind="l10n_href: z.string.url_support_camera_access_denied, l10n_text: z.string.warning_learn_more"></a>
```
Comment (lipis) [inline app/page/template/warning.htm]: `data-bind` >> `rel, target` :)

``` html
<a class="warning-bar-link" data-bind="l10n_href: z.string.url_support_camera_access_denied, l10n_text: z.string.warning_learn_more" rel="nofollow noopener noreferrer" target="_blank"></a>
```

**Pull request merged**